release-gradle/buildSrc/src/main/resources/templates/index.html.eco

```diff
@@ -0,0 +1,214 @@
+---
```

Comment:
Please extract the part of this page to separate file and include. Then update only small template.